portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,61 @@
   width: 650px;
 }
 
-#greeting-container {
+#fun-fact-container {
   margin-top: 20px;
```

Comment:
nit: The indentation in `#fun-fact-container` is different than the rest of the ones below. Can you figure out which indentation is correct in the style guide and make them consistent?

Reply:
Done